apis/cloud.redhat.com/v1alpha1/clowdenvironment_types.go

```diff
@@ -239,6 +239,17 @@ type ClowdEnvironmentStatus struct {
 	// Important: Run "make" to regenerate code after modifying this file
 	TargetNamespace string                  `json:"targetNamespace"`
 	Deployments     common.DeploymentStatus `json:"deployments"`
+	Apps            []AppStatus             `json:"apps,omitempty"`
+}
+
+type AppStatus struct {
```

Comment:
The name `AppStatus` is a bit odd to me -- you're not giving the status of an app :slightly_smiling_face: .  Maybe something like `AppInfo` or `AppListing`?